### PR TITLE
[CCP] Implemented interaction for treatment outcomes icons view

### PR DESCRIPTION
**Pull request by noranda**
This PR satisfies [JIRA 1882](https://jira.mitre.org/browse/ASCODCP-1882) and implements the interaction components of the Treatment Outcomes Icons view as shown in the [MVP Design](https://www.figma.com/proto/rLpubNAfrqV99exbLVFz5c/core-cancer-pilot?node-id=1615%3A8637&viewport=1282%2C847%2C0.260075&scaling=min-zoom). The user can now select a compared treatment which will update the icons chart as well as show details about how many more patients survive/die and side effects.

Some things to note:
- I changed the default treatment selections in order to highlight the icons view with the maximum options given the data available. This can be changed back at any time.
- The side effects will not change when the time scale changes, as the data we have isn't separated by time scale.

_Pull requests into Flux require the following. Submitter should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable (**N/A**) and :white_check_mark:. Reviewers should complete every item on the bulleted list when reviewing._

## Submitter:

**Running the Application**

- [x] Manually tested in Chrome (primary browser for testing)
- [x] Manually tested in IE/Safari (verify app loads and basic feature testing)
- [x] Manually tested in Chrome in iPad mode (at minimum, viewing patient record)

**Task**

- [x] Branch implements task as expected
- [ ] Key application features are updated as needed for task and tested (see: [Key Application Features](https://github.com/FluxNotes/flux/wiki/Key-Application-Features))
- [ ] Task specific definition of done (as described in the Jira task) is met, if applicable.

**Documentation**

- [x] This pull request describes why these changes were made
- [x] Recognizes any potential shortcomings/bugs in the description 
- [ ] Pre release script is updated on the [Pre Release Testing Script wiki page](https://github.com/FluxNotes/flux/wiki/Pre-Release-Testing-Script) (add/remove single features to test, do not run through full script)
- [ ] Update the list of key features with new features on the [Key Application Features wiki page](https://github.com/FluxNotes/flux/wiki/Key-Application-Features)
- [ ] Document any new APIs/extension points
- [ ] Additional technical components and libraries are documented and added to [wiki](https://github.com/FluxNotes/flux/wiki/About-Flux-Notes) as needed

**Code Quality**

- [x] Code style guide is followed (see: [Code Style Guide](https://github.com/FluxNotes/flux/wiki/JavaScript-and-HTML-Code-Style-Guide))
- [x] Code is commented

**Tests**

- [x] Existing tests passed
- [ ] Added backend tests for new functionality added

## Reviewers:

- Check that code is maintainable and reusable. Code reuses existing code and infrastructure where appropriate.
- Check the PR and code accomplishes the task's purpose.
- Check that new tests appropriately test the new code, including edge cases. Other existing tests pass.
- Verify Pull Request checklist is correctly completed by submitter.
- Try to break the code

### CODE REVIEW

File: src/mcode-pilot/components/TreatmentOptionsOutcomesIcons/TreatmentOptionsOutcomesIcons.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                    additionalNumSurvive={additionalNumSurvive}
+                    selectedIndex={selectedIndex + 1}
+                    treatment={treatment}
+                />
+
+                <div className="icons-interaction">
+                    <div className="interaction-selection interaction-included">
+                        <div
+                            className="treatment-name-group"
+                            onClick={() => this.selectIconsTreatment('alone', includedTreatmentName)}>
```
[the remark this answers is not in the document]
Comment (noranda) [reply src/mcode-pilot/components/TreatmentOptionsOutcomesIcons/TreatmentOptionsOutcomesIcons.js]: If the selection headers were defined in an accordion-type view, I'd agree with this, but they are denoted by a caret to the left, so there's not really a "default" selection. It feels weird to click on a treatment again to select something else.

Any preference @dreeves4321?

**Comment by noranda** [comment]
[KG1] : Added a max-width to the visualizer to prevent right-aligned buttons from floating away when view widths are large.

**Comment by noranda** [comment]
@rdingwell: This is by design (see the MVP). The "+" means that the treatment is in addition to the included treatments. I can see how this can be confusing though and we should probably have @dreeves4321 review this, but probably shouldn't hold up this PR.
